### PR TITLE
Get Requests for Builds, Pipelines, and jobs completed. 

### PR DESCRIPTION
**Pull request by nicolaifsf**
Builds - Filter by status flag is still buggy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
 > Screwdriver CLI
 
-## Usage
-
-```bash
-npm install screwdriver-client
-```
+## Building from source  
+1. Clone down the go-swagger library into github.com/go-swagger/go-swagger  
+2. `$ git checkout tags/0.5.0 in the go-swagger library`
```
Comment (nkatzman) [inline README.md:14]: Is this actually the command to run?

Comment (nicolaifsf) [inline README.md:14]: nope. should be `$ git checkout tags/0.5.0` in the go-swagger directory

**Comment by nkatzman** [comment]
What of this is auto-generated vs written?

**Comment by nicolaifsf** [comment]
The files under commands and main.go are all written. Auto-generated are the client and models dirs. I did do some bug fixing in the models and clients, but just things such as changing the way dates are stored from go Date objects to strings, as there were failures in unmarshaling into the json objects defined in models. Also just some path issues to go-openapi.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+					Action: command.JobsList,
+					Flags: idFlag,
+			},
+			{
+					Name: "builds-list",
+					Usage: "List Builds",
+					Action: command.BuildList,
+					Flags: buildFlags,
+			},
+	}
```
Comment (Filbird) [inline main.go:72]: Not urgent, but down the line perhaps refactor to using sub commands:

Ex:

```
app.Commands = []cli.Command {
        {
            Name: "pipelines",
            Usage: "Options for pipelines",
            Subcommands: []cli.Command {
                {
                    Name: "create",
                    Usage: "Create a new pipeline",
                    ArgsUsage: "<scmUrl> [configUrl]",
                    Action: cmd.PipelinesCreate,
                },
                {
                    Name: "list",
                    Usage: "List all pipelines",
                    Action: cmd.PipelinesList,
                },
            },
        },
        {
            Name: "jobs",
            Usage: "Options for jobs",
            Subcommands: []cli.Command {
                {
                    Name: "list",
                    Usage: "List all builds",
                    Flags: []cli.Flag {
                        cli.StringFlag{
                            Name: "pipelineId, p",
                            Usage: "Only show jobs that are for this pipeline",
                        },
                    },
                    Action: cmd.JobsList,
                },
            },
        },
```

File: commands/builds.go [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+		resp, err := sd.Default.V3.GetV3Builds(nil)
+		if err != nil {
+			return cli.ShowSubcommandHelp(c)	
+		}
+		resp = buildsFilterJobs(resp, c)
+		resp = buildsFilterStatus(resp, c)
+		formattedPrint(resp)
+	} else if len(c.Args()) == 1{
+		args := c.Args()	
+		params := v3.NewGetV3BuildsIDParams().WithID(args[0])
```
Comment (nkatzman) [inline commands/builds.go]: can you declare variables for these magic numbers? It'd help with readability of the code

**Pull request closed without merging**